src/main/java/racingcar/common/Utils.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.common;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import racingcar.game.user.User;
+
+import java.util.List;
+
+public class Utils {
+    public static int getRandomNumber() {
+        return Randoms.pickNumberInRange(Constant.MIN_RANDOM_RANGE, Constant.MAX_RANDOM_RANGE);
+    }
+
+    public static int findBestScore(final List<Integer> scores) {
+        int bestScore = -1;
+        for (final int targetScore : scores) {
+            if (targetScore > bestScore) {
+                bestScore = targetScore;
+            }
+        }
+        return bestScore;
+    }
+
+    public static List<User> findBestUser(List<User> users, final int bestScore) {
+        return users.stream().filter(user -> user.getCar().getPosition() == bestScore).toList();
+    }
```

Comment:
(가독성을 더 높여볼 수 있을 것 같아요!)

```suggestion
    public static List<User> findBestUser(List<User> users, final int bestScore) {
        return users.stream()
            .filter(user -> user.getCar().getPosition() == bestScore)
            .toList();
    }
```

스트림 사용 시 개행에 대한 관례가 있는 것은 아니지만,
스트림의 `중간 연산`, `최종 연산` 을 적절히 개행하는 것도 가독성이 괜찮더라고요 :)